File: src/peasoup.py
if __name__ == "__main__":
    n = int(input())
    flag = False
    while(n > 0):
        k = int(input())
        resname = input()
        menu = [input() for i in range(k)]
        if "pea soup" in menu and "pancakes" in menu:
            flag = True
            print(resname); break
        n -= 1
    if flag == False: print("Anywhere is fine I guess")

File: src/peasoup.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int n; cin >> n;
    bool bigflag = false;
    while(n--){
        int k; cin >> k;
        bool flag1, flag2 = false;
        cin.ignore();
        string resname; getline(cin, resname);
        vector<string> menu;
        for(int i = 0; i < k; i++){
            string men; getline(cin, men);
            menu.push_back(men);
        }
        auto it1 = find(menu.begin(), menu.end(), "pea soup");
        auto it2 = find(menu.begin(), menu.end(), "pancakes");
        if((it1 != menu.end()) && (it2 != menu.end())){
            bigflag = true;
            cout << resname;
            break;
        }
    }
    if(bigflag == false) cout << "Anywhere is fine I guess";
    return 0;
}